Add patch to fix hardcoded documentation
Index: robojournal/ui/mainwindow.cpp
===================================================================
--- robojournal.orig/ui/mainwindow.cpp	2013-10-22 11:56:07.328363672 +0530
+++ robojournal/ui/mainwindow.cpp	2013-10-22 11:56:42.164363280 +0530
@@ -1044,8 +1044,8 @@
         assistant="/usr/bin/assistant";
     }
 
-    compiled_help_path="/usr/share/doc/robojournal/robojournal.qch";
-    collection_path="/usr/share/doc/robojournal/robojournal.qhc";
+    compiled_help_path="/usr/share/doc/robojournal-doc/robojournal.qch";
+    collection_path="/usr/share/doc/robojournal-doc/robojournal.qhc";
 #endif
 
     QFile assistant_exec(assistant);
@@ -1109,7 +1109,7 @@
 #endif
 
 #ifdef unix
-                location="/usr/share/doc/robojournal";
+                location="/usr/share/doc/robojournal-doc";
 #endif
 
                 b.critical(this,"RoboJournal","RoboJournal could not locate the collection file"
